src/components/SearchForm/TermSelector.js

```diff
@@ -34,45 +35,9 @@ class TermSelector extends PureComponent {
-                    <MenuItem value={'2020 Spring'}>2020 Spring Quarter</MenuItem>
-                    <MenuItem value={'2020 Winter'}>2020 Winter Quarter</MenuItem>
-                    <MenuItem value={'2019 Fall'}>2019 Fall Quarter</MenuItem>
-                    <MenuItem value={'2019 Summer2'}>2019 Summer Session 2</MenuItem>
-                    <MenuItem value={'2019 Summer10wk'}>2019 10-wk Summer</MenuItem>
-                    <MenuItem value={'2019 Summer1'}>2019 Summer Session 1</MenuItem>
-                    <MenuItem value={'2019 Spring'}>2019 Spring Quarter</MenuItem>
-                    <MenuItem value={'2019 Winter'}>2019 Winter Quarter</MenuItem>
-                    <MenuItem value={'2018 Fall'}>2018 Fall Quarter</MenuItem>
-                    <MenuItem value={'2018 Summer2'}>2018 Summer Session 2</MenuItem>
-                    <MenuItem value={'2018 Summer10wk'}>2018 10-wk Summer</MenuItem>
-                    <MenuItem value={'2018 Summer1'}>2018 Summer Session 1</MenuItem>
-                    <MenuItem value={'2018 Spring'}>2018 Spring Quarter</MenuItem>
-                    <MenuItem value={'2018 Winter'}>2018 Winter Quarter</MenuItem>
-                    <MenuItem value={'2017 Fall'}>2017 Fall Quarter</MenuItem>
-                    <MenuItem value={'2017 Summer2'}>2017 Summer Session 2</MenuItem>
-                    <MenuItem value={'2017 Summer10wk'}>2017 10-wk Summer</MenuItem>
-                    <MenuItem value={'2017 Summer1'}>2017 Summer Session 1</MenuItem>
-                    <MenuItem value={'2017 Spring'}>2017 Spring Quarter</MenuItem>
-                    <MenuItem value={'2017 Winter'}>2017 Winter Quarter</MenuItem>
-                    <MenuItem value={'2016 Fall'}>2016 Fall Quarter</MenuItem>
-                    <MenuItem value={'2016 Summer2'}>2016 Summer Session 2</MenuItem>
-                    <MenuItem value={'2016 Summer10wk'}>2016 10-wk Summer</MenuItem>
-                    <MenuItem value={'2016 Summer1'}>2016 Summer Session 1</MenuItem>
-                    <MenuItem value={'2016 Spring'}>2016 Spring Quarter</MenuItem>
-                    <MenuItem value={'2016 Winter'}>2016 Winter Quarter</MenuItem>
-                    <MenuItem value={'2015 Fall'}>2015 Fall Quarter</MenuItem>
-                    <MenuItem value={'2015 Summer2'}>2015 Summer Session 2</MenuItem>
-                    <MenuItem value={'2015 Summer10wk'}>2015 10-wk Summer</MenuItem>
-                    <MenuItem value={'2015 Summer1'}>2015 Summer Session 1</MenuItem>
-                    <MenuItem value={'2015 Spring'}>2015 Spring Quarter</MenuItem>
-                    <MenuItem value={'2015 Winter'}>2015 Winter Quarter</MenuItem>
-                    <MenuItem value={'2014 Fall'}>2014 Fall Quarter</MenuItem>
+                    {termData.map((q) => (
+                        <MenuItem value={q.shortName}>{q.longName}</MenuItem>
+                    ))}
```

Comment:
Nit: use `term` instead of `q`. I know `q` is just shorthand for quarter, but it's a little more clear to just write out the whole word `term`
```suggestion
                    {termData.map((term) => (
                        <MenuItem value={term.shortName}>{term.longName}</MenuItem>
                    ))}
```

Reply:
Side note: I'm REALLY stoked to not have to write out these long `<MenuItem>` elements every again haha